connectors/v2/youtube.js

```diff
@@ -65,8 +65,18 @@ Connector.isStateChangeAllowed = function() {
 Connector.getArtistTrack = function () {
 	var text =$(Connector.artistTrackSelector).text();
 
-	text = text.replace(/^\[[^\]]+\]\s*-*\s*/i, ''); // remove [genre] from the beginning of the title
-	return Connector.splitArtistTrack(text);
+	// Remove [genre] from the beginning of the title
+	text = text.replace(/^\[[^\]]+\]\s*-*\s*/i, '');
+
+	let {artist, track} = Connector.splitArtistTrack(text);
+	if (artist === null && track === null) {
+		// Look for Artist "Track"
+		let artistTrack = text.match(/(.+?)\s"(.+?)"/);
+		if (artistTrack) {
+			[/* skip */, artist, track] = artistTrack;
```

Comment:
Could we assign differently? That skip comment implies to me that this could be done more clearer. Aside that looks good.